webapp/components/wpt-results.html

```diff
@@ -237,9 +237,11 @@
                 </template>
                 <template is="dom-if" if="[[diffShown]]">
                   <td class$="numbers [[ testResultClass(node, index, diffRun, 'passes') ]]">
-                    <span class$="passes [[ testResultClass(node, index, diffRun, 'passes') ]]">{{ getNodeResultDataByPropertyName(node, -1, diffRun, 'passes') }}</span>
+                    <span class="delta passes">{{ getNodeResultDataByPropertyName(node, -1, diffRun, 'passes') }}</span>
```

Comment:
Commenting on a random changed line in this file. I think that only you and I will know what the 3 numbers in the view means. Can you add some legend under the diff icon, maybe where hovering shows extra information like for runs?

Reply:
Maybe in a follow-up; let's improve cosmetics separately.